src/net.rs

```diff
@@ -628,6 +628,74 @@ pub fn dev_status() -> ProcResult<HashMap<String, DeviceStatus>> {
     Ok(map)
 }
 
+/// An entry in the ipv4 route table
+#[derive(Debug, Clone)]
+pub struct RouteEntry {
+    /// Interface to which packets for this route will be sent
+    pub iface: String,
+    /// The destination network or destination host
+    pub destination: Ipv4Addr,
+    pub gateway: Ipv4Addr,
+    pub flags: u16,
+    /// Number of references to this route
+    pub refcnt: u16,
+    /// Count of lookups for the route
+    pub r#use: u16,
```

Comment:
Can you rename this to something like `count` or `in_use`?  I think it would be annoying to consumers of this field to have to use a raw identifier for this field.

Reply:
Ok, I rename it to `use_` ident

Reply:
Sorry, but unfortunately I don't really like `use_` either, since trailing underscores aren't part of normal rust style

Reply:
I rename it to `in_use`